HW05/ivan_shklovski/hw_02_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative './hw02'
+
+using NewArrayMethods
+
+describe 'my_each' do
+  let!(:arr) { [1, 2, 3, 4, 5, 6, 7, 8] }
```

Comment:
Why do you need `let!` here?
```suggestion
  let(:arr) { [1, 2, 3, 4, 5, 6, 7, 8] }
```